src/qt/bitcoin.cpp

```diff
@@ -659,7 +667,7 @@ int main(int argc, char *argv[])
     // Re-initialize translations after changing application name (language in network-specific settings can be different)
     initTranslations(qtTranslatorBase, qtTranslator, translatorBase, translator);
 
-#ifdef ENABLE_WALLET
+#if defined(ENABLE_WALLET) && defined(ENABLE_BIP70)
```

Comment:
Won't this break opening non-BIP70 `bitcoin:` URIs?

Reply:
Yes, as a by-effect this currently removes `bitcoin:` URL support too.
(it probably shouldn't)

Reply:
That would break opening links from a browser or a mail client.